extensions/wikia/ContentReview/scripts/contentReviewTestMode.js

```diff
@@ -18,8 +19,13 @@ define(
 	}
 
 	function bindEvents() {
-		$('.content-review-test-mode-enable').on('click', enableTestMode);
-		$('#WikiaPage').on('click', '.content-review-test-mode-disable', disableTestMode);
+		if (win.skin === 'monobook') {
+			$('body').on('click', '.content-review-module-test-mode-enable', enableTestMode);
```

Comment:
Ok, so next question - why can't you use the same `body` bind for Oasis and Monobook? :)